storage/mysql/log_storage.go

```diff
@@ -16,20 +16,22 @@ import (
 	"github.com/google/trillian/merkle"
 	"github.com/google/trillian/storage"
 	"github.com/google/trillian/storage/cache"
+	"time"
```

Comment:
Move this up with the other standard library imports.

https://github.com/golang/go/wiki/CodeReviewComments#imports